output.tf

```diff
@@ -0,0 +1,3 @@
+output "gitlab_project_id" {
+  value = gitlab_project.this[0].id
```

Comment:
this may fail if create=false,
maybe you can use try(gitlab_project.this[0].id, null) construction?